src/chunk.zig

```diff
@@ -173,6 +173,20 @@ pub const ChunkPosition = struct { // MARK: ChunkPosition
 	wz: i32,
 	voxelSize: u31,
 
+	pub const HashContext = struct {
+		pub fn hash(_: HashContext, a: ChunkPosition) u64 {
+			var h = std.hash.Wyhash.init(0);
+			h.update(std.mem.asBytes(&a.wx));
+			h.update(std.mem.asBytes(&a.wy));
+			h.update(std.mem.asBytes(&a.wz));
+			h.update(std.mem.asBytes(&a.voxelSize));
+			return h.final();
+		}
+		pub fn eql(_: HashContext, a: ChunkPosition, b: ChunkPosition) bool {
```

Comment:
If you don't use a the HashContext, why add a new namespace for it?
Also both of these functions exist already, so either add a context to those, or call them internally so we don't have two different implementations of everything.

Reply:
Right, lets use list anyway. 
Different topic: what do you think about that Wyhash thingy? I found it in AutoHashMap implementation and it looks really universal.

Reply:
I don't know, I'd need to take a look inside to figure out if it's actually any useful.
It might be doing too much slow stuff, I know that's true for Zig's random number generators.